test/e2e/specs/site-editor/site-editor-url-navigation.spec.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Site editor url navigation', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'emptytheme' );
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'twentytwentyone' );
+	} );
+
+	test.describe(
+		'Redirection after template and template part creation',
+		() => {
+			test.afterEach( async ( { requestUtils } ) => {
+				await Promise.all( [
+					requestUtils.deleteAllPosts(),
+					requestUtils.deleteAllTemplates( 'wp_template' ),
+					requestUtils.deleteAllTemplates( 'wp_template_part' ),
+				] );
+			} );
+			test( 'Redirection after template creation', async ( {
+				admin,
+				page,
+				requestUtils,
+			} ) => {
+				await requestUtils.createPost( {
+					title: 'Demo',
+					status: 'publish',
+				} );
+				await admin.visitSiteEditor();
+				// We need to wait the response from the `posts` request in order to click the augmented menu item,
+				// that includes the options for creating templates for specific posts.
+				await Promise.all( [
+					page.waitForResponse(
+						( resp ) =>
+							resp
+								.url()
+								.includes(
+									'/index.php?rest_route=%2Fwp%2Fv2%2Fposts&context=view'
+								) && resp.status() === 200
+					),
+					page.click( 'role=button[name="Templates"]' ),
+				] );
+				await page.click( 'role=button[name="Add New Template"i]' );
+				await page
+					.getByRole( 'menuitem', {
+						name: /Single item: Post/,
```

Comment:
Oops, I mean the original was the same as string literal:

```suggestion
						name: 'Single item: Post',
```

By using the regex syntax we are effectively opting out of the default text-matching behavior.

Reply:
Got it. I updated the rest usages too.